### PR TITLE
Add configs for windows packages and detect ip to dcos-config-win

### PR DESCRIPTION
**Pull request by KostiantynOleksiienko**
## High-level description

This pr is adding configs to dcos-config-win.yaml for dcos-diagnostics, telegraf and add detect ip section.

## Corresponding DC/OS tickets (required)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS_OSS-5650) JIRA title / short description.

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS_OSS-5650) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot changelog-not-required

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot request-review @DominikDary

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot request-review @DominikDary

**Comment by DominikDary** [comment]
@jongiddy please review as well

File: gen/dcos-config-win.yaml [hunk 1]
```diff
@@ -7,3 +7,209 @@ package:
...
+              Measurement = "dcos_diagnostics"
+
+
+      [[inputs.docker]]
+         endpoint = "ENV"
+      
+      [[inputs.mesos]]
+      
+        timeout = 100
+         slaves = ["http://{{ local_ip }}:5051"]
```
Comment (jongiddy) [inline gen/dcos-config-win.yaml]: This is not provided as a variable currently. Where will this come from?

Comment (KostiantynOleksiienko) [reply gen/dcos-config-win.yaml]: This is copied from telegraf wich used on winagent by Oleksandr

**Comment by jongiddy** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by orsenthil** [comment]
Shoudl we rebase and test this?

It seems to be failing the train PR: https://github.com/dcos/dcos/pull/6541

**Comment by orsenthil** [comment]
@mesosphere-mergebot label work in progress

**Comment by orsenthil** [comment]
Please set to ready for review once this change is rebased against the master changes.

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by DominikDary** [comment]
Build is failing, suggesting to rebase this to latest master @KostiantynOleksiienko

**Comment by KostiantynOleksiienko** [comment]
@DominikDary I have updated branch

File: gen/dcos-config-windows.yaml [hunk 2]
```diff
@@ -15,3 +15,200 @@ package:
...
+  - path: C:\dcos\opt\etc\check_time.env
+    content: |
+        ENABLE_CHECK_TIME=true
+
+  # Config for dcos-diagnostics
+  - path: C:\dcos\opt\etc\dcos-diagnostics-config.json
+    content: |
+      {
+        "role": "agent",
+        "endpoint-config": "C:\dcos\opt\etc\dcos-diagnostics-endpoint-config.json",
```
Comment (jongiddy) [inline gen/dcos-config-windows.yaml]: In JSON, backslashes need to be doubled

Comment (KostiantynOleksiienko) [reply gen/dcos-config-windows.yaml]: I have changed to double backslashes.

**Pull request merged**